src/store/home/home.thunks.js

```diff
@@ -143,18 +143,20 @@ function refreshAccounts (hermezEthereumAddress, poolTransactions, pendingDeposi
         undefined,
         axiosConfig
       )
-      const requests = accountsTask.data.fromItemHistory
-        .reduce((requests, fromItem) => ([
-          ...requests,
-          CoordinatorAPI.getAccounts(
-            hermezEthereumAddress,
-            undefined,
-            fromItem,
-            undefined,
-            undefined,
-            axiosConfig
-          )
-        ]), [initialReq])
+
+      const requests = accountsTask.status === 'reloading' ? [] : accountsTask.data.fromItemHistory
```

Comment:
This doesn't fix the root problem :) The problem looks that when `status` is `reloading`, `fromItemHistory` is not inside the `data` property, and it should.

Reply:
Fixed